### DISCUSSION TITLE
sever selection menu

### DISCUSSION THREAD

**Discussion by inzi2u** [ABJC/ABJC-tvOS#44] [Q&A]
i am unable to log in to my jellyfin server. when pressing continue after entering the details, i get an authfailed.
i know the password and username are correct.
I would like to know what exactly needs to be entered in the 3 spaces on the server selection screen.
My server is a remote server behind a nginx reverse proxy .
Domain https://jellyfin.xxxxx.yyyy connects to the webui fine when i try via a browser

what i entered in the fields were 
1. jellyfin
2. 443
3. https://jellyfin.xxxxx.yyyy
4. https On

still get an auth failed

**Comment by Ribero** [comment]
Facing the same issue, unable to login. 
Works fine when connecting from web browser or other client. 
Jellyfin server is behind a nginx reverse proxy.
